### PR TITLE
fix opensea crash

### PR DESCRIPTION
**Pull request by skylarbarrera**

### CODE REVIEW

**Review by jinchung — APPROVED** [review]
🍠

File: src/helpers/assets.js [hunk 1]
```diff
@@ -184,7 +184,7 @@ export const buildUniqueTokenList = (uniqueTokens, selectedShowcaseTokens) => {
   const showcaseTokens = [];
   const bundledShowcaseTokens = [];
 
-  const grouped = groupBy(uniqueTokens, token => token.collection.name);
+  const grouped = groupBy(uniqueTokens, token => token?.collection?.name);
```
Comment (jinchung) [inline src/helpers/assets.js:187]: so what happens when a token doesn't have a collection name?

Comment (osdnk) [reply src/helpers/assets.js:187]: I think this is a valid concern. We don't have this optional chaining on the web!

Comment (skylarbarrera) [reply src/helpers/assets.js]: @osdnk was an issue with previous local storage not having collection data. Reverted and bumped data version 💯

**Review by jinchung — APPROVED** [review]
🌮

**Pull request merged**